NEWS.md

```diff
@@ -1,3 +1,18 @@
+# scoringutils 1.1.5
+
+## Feature updates
+- Added a new function, `set_forecast_unit()` that allows the user to set the 
+forecast unit manually. The function removes all columns that are not relevant
+for uniquely identifying a single forecast. If not done manually, `scoringutils` 
+attempts to determine the unit
```

Comment:
some weird linebreak stuff showing here for me?

Reply:
weird.. I removed some linebreaks, so should be fixed hopefully.